charms/jupyter-controller/contributing_inputs.yaml

```diff
@@ -0,0 +1,19 @@
+charm: "jupyter_controller"
+tox_example_dev_env: "lint"
+os_base: "ubuntu-20.04"
+deploy_opts: |-
+  \
+    --resource oci-image=$(yq '.resources."oci-image"."upstream-source"' metadata.yaml)
+tox_example_test_env: "test"
+charm_specific_text: |-
+
+
+  ## Updating the charm for new versions of the workload
+
+  To upgrade the source and resources of this charm, you must:
+  
+  1. Bump the `oci-image` in `metadata.yaml`
+  1. Update the charm source for any changes, such as:
+      - YAML manifests in `src/` and/or any Kubernetes resource in `pod_spec`
+      - New or changed configurations passed to pebble workloads or through `pod.set_spec`
+  1. Ensure integration and unit tests are passing; fix/adapt them otherwise
```

Comment:
Let's add a new line at the end of the file, please.

Reply:
Done.